distribution/lib/Standard/Base/0.2.32-SNAPSHOT/src/Data/Vector.enso

```diff
@@ -546,7 +548,7 @@ type Vector
              [1, 2, 3, 4, 5].drop_start 1
     drop_start : Integer -> Vector Any
     drop_start count = if count >= this.length then here.new 0 (x -> x) else
-        here.new (this.length - count) (i -> this.unsafe_at i+count)
+        Vector (Slice this.store count (this.length - count))
```

Comment:
I'd suggest at least adding a TODO somewhere so that we don't forget that we can and at some point definitely should ensure that this is a flat Slice instead of a possibly long stack of slices.

I'd also suggest adding a benchmark which first does ~100 drops and then calls `at` many times, so that we can have the performance of at after many slices under control. (In the current version it may actually be not very fast due to a lot of indirection happening in the stack of Slices).

Reply:
+1 to this